cves/kernel/CVE-2017-6074.yml

```diff
@@ -466,12 +505,10 @@ CWE_instructions: |
     CWE: 123            # also ok
 CWE:
 - 415
-CWE_note: |
-  CWE as registered in the NVD. If you are curating, check that this
-  is correct and replace this comment with "Manually confirmed".
+CWE_note: Manually confirmed
 nickname_instructions: |
   A catchy name for this vulnerability that would draw attention it.
   If the report mentions a nickname, use that.
   Must be under 30 characters. Optional.
-nickname:
+nickname: DCCP use-after-free
```

Comment:
Eh - not very understandable or engaging. How about something like "Congestion Corruption"